### PR TITLE
Feature/541 implement 404 page

### PR DESCRIPTION
**Pull request by dkrakau**
### Description

ISSUE: #541 implement 404 page

## Checklist

- [x] My code follows the style guidelines
- [ ] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [ ] The Pull Request covers all the Checkpoints from the tagged Issue
- [ ] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [x] No tests required
- [ ] Yes
- [ ] No

### CODE REVIEW

File: frontend/src/pages/NotFoundPage.module.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  width: 10rem;
+  color: var(--ion-color-secondary);
+}
+
+.errorCode {
+  margin: 0;
+  text-align: center;
+  font-size: 4rem;
+  font-weight: 400;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (AconexOfficial) [inline frontend/src/pages/NotFoundPage.module.css:20]: Is there a specific reason to use a specific different font than everywhere else?

File: frontend/src/pages/NotFoundPage.module.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  color: var(--ion-color-medium-shade);
+}
+
+.errorMessage {
+  text-align: center;
+  font-size: 2rem;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (AconexOfficial) [inline frontend/src/pages/NotFoundPage.module.css:27]: Same as first comment

File: frontend/src/pages/NotFoundPage.module.css (hunk 1 above)
Comment (dkrakau) [reply frontend/src/pages/NotFoundPage.module.css]: Font removed.

File: frontend/src/pages/NotFoundPage.module.css (hunk 2 above)
Comment (dkrakau) [reply frontend/src/pages/NotFoundPage.module.css]: Font removed.

**Review by AconexOfficial — APPROVED** [review]

**Pull request merged**